sky/clouds/service_catalog/kubernetes_catalog.py

```diff
@@ -46,38 +47,104 @@ def list_accelerators(
         case_sensitive: bool = True,
         all_regions: bool = False,
         require_price: bool = True) -> Dict[str, List[common.InstanceTypeInfo]]:
+    return list_accelerators_realtime(gpus_only, name_filter, region_filter,
+                                      quantity_filter, case_sensitive,
+                                      all_regions, require_price)[0]
```

Comment:
That's a good point.. the overhead compared to a the previous implementation isn't much different since the previous implementation was also invoking the kubernetes API:

```
This branch:
multitime -n 5 sky launch --dryrun -y --gpus T4:1
===> multitime results
1: sky launch --dryrun -y --gpus T4:1
            Mean        Std.Dev.    Min         Median      Max
real        3.883       0.064       3.782       3.883       3.982
user        2.775       0.081       2.654       2.766       2.871
sys         3.136       0.285       2.676       3.268       3.448


Master: 
multitime -n 5 sky launch --dryrun -y --gpus T4:1
1: sky launch --dryrun -y --gpus T4:1
            Mean        Std.Dev.    Min         Median      Max
real        3.863       0.032       3.829       3.860       3.917
user        2.713       0.023       2.670       2.716       2.735
sys         3.438       0.097       3.267       3.471       3.535
```

That said, we should put a lru cache with a time-to-live (TTL) to expire based on time. Added a TODO.